ci/kokoro/windows/continuous-win-2019.cfg

```diff
@@ -0,0 +1,30 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+build_file: "google-cloud-cpp/ci/kokoro/windows/build.bat"
+timeout_mins: 120
+
+gfile_resources: "/bigstore/cloud-cpp-integration-secrets/build-results-service-account.json"
+
+action {
+  define_artifacts {
+    regex: "**/win_minidumps/*.dmp"
+  }
+}
+
+env_vars {
+  key: "BUILD_CACHE"
+  value: "gs://cloud-cpp-kokoro-results/build-artifacts/vcpkg-installed-win-2019.zip"
```

Comment:
Do we need a similar change for the `-presubmit` build?

Reply:
Added presubmit config too